package.json

```diff
@@ -3,7 +3,7 @@
     "build": "npm run lint && npm run clean && npm run webpack",
     "webpack": "webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
     "postbuild": "npm run package",
-    "package": "tfx extension create --rev-version",
+    "package": "tfx extension create --rev-version --manifest-globs .\\mimeo-vsts-extensions.json",
```

Comment:
Would it make more sense to make this `./` to be consistent?

Reply:
yeah, good catch! I had originally just copied this from the README without thinking about it, but I can flip that backslash around really quick